### PR TITLE
awscli-ssm-plugin 1.1.54.0 (new formula)

### PR DESCRIPTION
**Pull request by rclaveau-tech**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/awscli-ssm-plugin.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class AwscliSsmPlugin < Formula
+  include Language::Python::Virtualenv
+
+  desc "Session Manager plugin for Amazon AWS command-line interface"
+  homepage "https://aws.amazon.com/systems-manager/"
+  url "https://s3.amazonaws.com/session-manager-downloads/plugin/1.1.54.0/mac/sessionmanager-bundle.zip"
```
Comment (SMillerDev) [inline Formula/awscli-ssm-plugin.rb]: This looks macOS specific. Formula need to build from source and can't contained os specific binaries.

**Review by SMillerDev — COMMENTED** [review]
Please check this.

File: Formula/awscli-ssm-plugin.rb (hunk 1 above)
Comment (sousmangoosta) [reply Formula/awscli-ssm-plugin.rb]: Thanks @SMillerDev , let me check

Comment (sousmangoosta) [reply Formula/awscli-ssm-plugin.rb]: Ok, didn't see there was a Cask for it, sorry for the noise 😢

**Comment by sousmangoosta** [comment]
Os specific binaries not allowed here

**Pull request closed without merging**